examples/multigpu_advtrain/cifar_input.py

```diff
@@ -0,0 +1,180 @@
+"""
```

Comment:
we already have a cleverhans/utils/utils_mnist.py, maybe this should be cleverhans/utils/utils_cifar.py

Reply:
Fixed in 1fcb2a4.